### ISSUE TITLE
Link some instructions how to build binary wheels with C code?

### ISSUE THREAD

**Issue by techtonik** [meshy/pythonwheels#59]
Currently, it is not evident where to send the people who want to wrap their extensions into package, or (maybe even more importan) port their eggs.

**Comment by meshy** [comment]
You're right, it could do with a little more direction. I was thinking about adding a link to the PyPA guide to [Packaging your Project](https://packaging.python.org/en/latest/distributing.html#packaging-your-project). Does that seem appropriate to you?

**Comment by techtonik** [comment]
I don't have time to follow the instruction right, because I am on Windows and don't have compiler installed. But the good proof check would it https://github.com/frida/frida-python/issues/38 can be solved by a person who care little about wheel. The key points:

* if he could directly understand what needs to be done and what should be available (setup.py?) without bouncing from the site (usually it is scanning page and selecting info in about 0.1-3 seconds)
* complete the task in 15 minutes

**Comment by meshy** [comment]
I don't think that we would be able to provide the information required to compile a project like that. Anyone with that sort of project will surely have to use the official docs instead.

**Comment by techtonik** [comment]
pythonwheels.com is of course, not a place to this information, but it should link to the tutorial that is most helpful. In most technical projects, official docs are usually a place of last resort.

**Comment by webknjaz** [comment]
https://packaging.python.org/en/latest/guides/packaging-binary-extensions/
